packages/sit-onyx/src/components/OnyxButton/OnyxButton.vue

```diff
@@ -185,6 +195,22 @@ const emit = defineEmits<{
       line-height: 1.5rem;
     }
 
+    $p: &;
+
+    &--ripple-animation {
+      &:hover:enabled {
+        --onyx-button-background-color: var(--onyx-button-background-hover-color);
+      }
+      #{$p}__label,
+      #{$p}__icon {
+        position: relative;
+        pointer-events: none;
```

Comment:
When using `pointer-events: none` on the label, it is no longer markable/copyable via mouse...do you know if there is a way to get rid of this? Maybe by using the click handler on the parent?

Reply:
ok now I'm exposing the event handlers to the parent - so the Button in this case. And the parent registers them.
Not sure if there's a nicer way ...